LibSerialize.lua

```diff
@@ -1310,11 +1478,8 @@ function LibSerialize:IsSerializableType(...)
     return serializeTester:_CanSerialize(canSerializeFnOptions, ...)
 end
 
-function LibSerialize:SerializeEx(opts, ...)
-    local ser = CreateSerializer(opts)
-
+local function serializeOperation(ser, ...)
```

Comment:
This and the deserialization variant should be moved to where `CreateSerializer`/etc. are defined - this part of the file is just for the public APIs.